src/DroneSafetyApi/Models/IDataPointRepository.cs

```diff
@@ -6,10 +6,10 @@
 
 namespace DroneSafetyApi.Models
 {
-    public interface IHazardsRepository
+    public interface IDataPointRepository
     {
         // Keys in the return value signify the names of each relevant data source
         // Values in the return value signify the hazards intersecting in the given area
-        Dictionary<string, IEnumerable<Hazard>> GetHazardsOverlappingWith(Polygon area);
+        IEnumerable<DataPoint> GetDataPoints(double x, double y);
```

Comment:
What you you mean by "GetDataPoints"? Would be nice to have a slightly more descriptive method name.